Contributing.md

```diff
@@ -0,0 +1,31 @@
+Licensing
+=========
+
+The code is distributed under the BSD 2-clause license. Contributors making pull requests must agree that they are able and willing to put their contributions under that license.
```

Comment:
I'd prefer to keep lines broken at ~80 chars also in md/rst files.

Reply:
Done!